odoo/modules/loading.py

```diff
@@ -97,7 +98,7 @@ def load_demo(env, package, idref, mode):
         Failure = env.get('ir.demo_failure')
         if todo and Failure is not None:
             todo.state = 'open'
-            Failure.create({'module_id': package.id, 'error': str(e)})
```

Comment:
Make it a separate PR?

Reply:
Done here thanks https://github.com/odoo/odoo/pull/189259 in 16 instead
I wasn't sure this change would be acceptable so I was expecting to first check with your team first and I then forgot :see_no_evil: